src/hooks/useENS.ts

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "ethers";
```

Comment:
would be great to keep using web3 instead, unless you dont have access to similar functions

Reply:
@ethedev web3 dont have support for ens resolving 😢